extensions/websockets-next/runtime/src/main/java/io/quarkus/websockets/next/WebSocketConnector.java

```diff
@@ -132,4 +135,16 @@ default WebSocketClientConnection connectAndAwait() {
         return connect().await().indefinitely();
     }
 
+    /**
+     * This method allows to customize {@link WebSocketConnectOptions} and {@link WebSocketClientOptions}.
+     * Connection options configured directly with this API, such as host and port, have higher priority.
```

Comment:
```suggestion
     * Connection options configured directly with this API, such as host and port, have higher priority than this customizer.
```

Because "this API" may be a bit confusing...